deploy/charts/dubbo-admin/values.yaml

```diff
@@ -79,6 +52,119 @@ imagePullSecrets: []
 # - name: secretName
 
 
+##  @param Admin Default Enable Configuration
+dubbo:
```

Comment:
I think we need to discuss whether it's better to put all admin parameters inside values.yaml like in this pull request or better to keep a separate configuration file.